src/node/app.ts

```diff
@@ -44,7 +44,7 @@ const listen = (server: http.Server, { host, port, socket }: ListenOptions) => {
       server.listen(socket, onListen)
     } else {
       // [] is the correct format when using :: but Node errors with them.
-      server.listen(port, host.replace(/^\[|\]$/g, ""), onListen)
+      server.listen(parseInt(port, 10), host.replace(/^\[|\]$/g, ""), onListen)
```

Comment:
❓ Guessing this is to please TypeScript and sure port is indeed a `number`?

Reply:
Indeed, upstream anticipates a stringly number, I’m guessing because their server arguments interface isn’t as well defined as their desktop variant.